### PR TITLE
Now we can track the amount of wins

### PR DESCRIPTION
**Pull request by farhaannishtar**
Created two hooks to keep track of the amount of win for X and O.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -32,14 +35,19 @@ function App() {
...
+        return '';
+    }  
   }
 
   const resetGame = () => {
+    if (winner === 'X') {
+      setXWins(xWins + 1);
+    } else if (winner === 'O') {
+      setOWins(oWins + 1);
+    }
```
Comment (ahsojar) [inline src/App.tsx:50]: This is great! If you're feeling up for a challenge, I'm curious to see if you can get the score to update right when the winning happens, rather then when the user clicks 'reset game'. This will help you learn a new react concept called [`useEffect`](https://www.benmvp.com/blog/object-array-dependencies-react-useEffect-hook/).

If you move [lines 46-50](https://github.com/farhaannishtar/react-tic-tac-toe/pull/15/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R46-R50) outside of the resetGame function, what happens?

You'll probably notice that the counter increments much more than expected (it adds 2-3 points intead of just one). This is because, when we move the code outside of resetGame, it executes everytime the App component is re-rendered. As a reminder, the app rerenders anytime the props change, or any of the state values change-- so in our case, anytime the user clicks a square, the state changes, and causes a rerender.

However, in our specific example, we only care about upding the points when one specific state value changes: the `winner`. React's `useEffect` hook can help us with this. Here's how it works:

```
React.useEffect(() => {
  setScore()
}, [winner]); // Only re-run the effect if winner changes
```

Basically, useEffect takes in two parameters. the first is a `function` of the code you want to execute. The second is a `array` of dependent variables. The function will only rerun if the dependent variables change. (There could be multiple here...so if we did `[winner, xIsNext]` then the function would execute anytime 1) xIsNext chnaged value OR 2) the winner changed value).

Hopefully this is a helpful start to see if you can use useEffect to do what you want. Feel free to also merge this PR as is and try it out in a separate branch/PR.
